src/scripts/main.js

```diff
@@ -357,4 +357,31 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
+const table = document.querySelector('.dashboard');
+
+for (const i of people) {
+  const row = document.createElement('tr');
+  const tdName = document.createElement('td');
+  const tdGender = document.createElement('td');
+  const tdBorn = document.createElement('td');
+  const tdDied = document.createElement('td');
+  const tdAge = document.createElement('td');
+  const tdCentury = document.createElement('td');
+
+  tdName.innerText = i.name;
+  row.appendChild(tdName);
+  tdGender.innerHTML = i.sex;
```

Comment:
Consider using `innerText` instead of `innerHTML` to set the text content for `tdGender` to avoid potential security issues related to HTML injection.